Sources/GDPR/ViewModels/GDPRService.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public struct GDPRService {
+    var id: String
+    var name: String
+    var description: String
+    var supportDeletion: Bool?
```

Comment:
`var supportDeletion: Bool`
Optional is not needed. Will be initiated with default value `false`